packages/webcert/src/components/modal/CareProviderModal.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import FocusTrap from 'focus-trap-react'
+import { useDispatch, useSelector } from 'react-redux'
+import { getUser } from '../../store/user/userSelectors'
+import { setUnit } from '../../store/user/userActions'
+import { CareProviders } from './CareProviders'
+import { User } from '@frontend/common'
+
+const ModalContentWrapper = styled.div`
+  p + p {
+    margin-top: 0.25em !important;
+  }
+`
+
+const WrapText = styled.div`
+  white-space: normal;
+`
+
+interface Props {
+  title: string
+}
+
+const CareProviderModal: React.FC<Props> = ({ title }) => {
+  const dispatch = useDispatch()
+  const user = useSelector(getUser)
+  const [isOpen, setIsOpen] = useState(false)
+
+  const handleChooseUnit = (event: React.MouseEvent) => {
+    const unitId = event.currentTarget.id
+
+    dispatch(setUnit(unitId))
+    setIsOpen(false)
+  }
+
+  useEffect(() => {
```

Comment:
Is the local variable open necessary or can the actual check if the unit is set be used instead?